app/views/projects/projects/show.html.erb

```diff
@@ -1,21 +1,85 @@
-<h1><%= @project.title %></h1>
+<div class="row">
+<nav class="projects-nav">
+  <div class="projects-nav__title">
+    <h1><%= @project.title %></h1>
+  </div>
+  <div class="project-nav__menu">
+    <ul class="inline-list">
+      <li>
+        <a href="#">Projects home</a>
+      </li>
+      <li>
+        <a href="#">Tasks</a>
+      </li>
+      <li>
+        <a href="#">Message project owner</a>
+      </li>
+    </ul>
+  </div>
+</nav> 
+</div>     
 
-<div>
-  <%= sanitize @project.briefing %>
-</div>
+<div class="inner-canvas">
+  <div class="inner-canvas-header">
+   <div class="row">
+      <h1><%= @project.title %></h1>
+    </div>
+  </div>
 
-<aside>
-  <h1><%= @project.owner.name %></h1>
+  <div class="inner-canvas-body">
+    <div class="row">
+      <div class="project-body">
+        <div class="project-briefing">
+          <%= sanitize @project.briefing %>
+          <p class="project-meta">16 contributors</p>
+          <div class="project-workload">
+            <h3 class="project-workload-title">Workload</h3>
```

Comment:
What's the reason for using "Workload" over "Progress"?

Reply:
We decided to change this as 'progress' is too hard to measure, I can't quite remember the details but it was to do with how requests would flow through the tasks so progress might be misleading, whereas workload shows how much work is to be done and where effort is needed.